src/components/roles/NewRoleDialog.vue

```diff
@@ -3,7 +3,10 @@
     <v-card class="pa-4">
       <h2>New role</h2>
       <validation-observer ref="form" v-slot="{ invalid, handleSubmit }">
-        <form @submit.prevent="handleSubmit(publishRole)">
+        <form
+          @submit.prevent="handleSubmit(publishRole)"
+          @keydown.enter.prevent
```

Comment:
This solution fixes issue #53, but now users cannot add role responsibilities anymore by pressing enter. This is easily fixed by replacing `@keydown` with `@keypress`, because `@keypress` is the command used (in line 63) to add new responsibilities.
```suggestion
          @keypress.enter.prevent
```